pages/pil/create/views/index.jsx

```diff
@@ -18,19 +17,12 @@ const Index = ({ establishment, pilApplication, profile }) => (
           {
             links.map(link => (
               <li key={link}>
-                <h2>
-                  <Link
-                    page="pil.application"
-                    profile={profile.id}
-                    pil={pilApplication.id}
-                    label={<Snippet>{`${link}.title`}</Snippet>}
-                  />
-                </h2>
+                <h2><Snippet>{`${link}.title`}</Snippet></h2>
                 <p>
                   <Snippet>{`${link}.subtitle`}</Snippet>
                 </p>
-                <LinkButton type="button" href={`/e/${establishment.id}/people/${profile.id}/pil/${pilApplication.id}`}>
-                  <Snippet>pil.buttons.applyNow</Snippet>
+                <LinkButton type="button" href={`${url}?action=${link}`}>
```

Comment:
If you just want "the current url with a query string" then `?foo=bar` as an href attribute should do.

Reply:
clever girl